test/service/test_response.py

```diff
@@ -3865,3 +3870,55 @@ def test_file_urls(self):
             file = self.get_file('c343a47d-683f-571d-99c4-1331841b4e63')
             self.assertIsNone(file['url'])
             self.assertIsNone(file['drs_uri'])
+
+    def test_projects_response(self):
+        """
+        Verify the project response fields from a DCP2 canned bundle.
+        """
+        params = {
+            'catalog': self.catalog,
+            'filters': json.dumps({
+                'projectId': {
+                    'is': ['edc54c5d-8240-4681-8444-62a086d1d1be']
+                }
+            })
+        }
+        url = self.base_url.set(path='/index/projects', args=params)
+        response = requests.get(str(url))
+        response.raise_for_status()
+        response_json = response.json()
+        project = one(one(response_json['hits'])['projects'])
+        self.assertEqual('MADummyProject', project['projectShortname'])
+        expected_title = ('MA Test - Intrarenal Single-Cell Sequencing of '
+                          'Hepatitis B Virus Associated Membranous Nephropathy')
+        self.assertEqual(expected_title, project['projectTitle'])
+        expected_laboratory = [
+            'Department of Hematology',
+            'Department of Nephrology',
+            'Department of Organ Transplantation',
+            'Department of Pathology'
+        ]
+        self.assertEqual(expected_laboratory, project['laboratory'])
+        self.assertEqual(14, len(project['contributors']))
```

Comment:
```suggestion
        self.assertEqual(14, len(project['contributors']))
```

If we assert all other properties literally, why not this one?

Generally speaking, it seems a bit tedious to assert each property individually. Why not just one big dictionary comparison?